modules/Bio/EnsEMBL/Compara/PipeConfig/CITest/BuildCITestMasterDatabase_conf.pm

```diff
@@ -94,8 +92,8 @@ sub pipeline_create_commands {
         'cp ' . $self->o('java_hc_db_prop') . ' ' . $self->o('backups_dir') . '/database.defaults.properties',
```

Comment:
I guess you won't need that when you move to @muffato's new wrapper script for running HCs

Reply:
Yes, this has been addressed in that PR.